### PR TITLE
wm-timemons decomped

### PR DESCRIPTION
**Pull request by MokhaLeee**

### CODE REVIEW

File: src/worldmap_timemons.c [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+            break;
+
+        case CHAPTER_MODE_EPHRAIM:
+            lut1 = WmMonsterGenerateRates_XmapEphraim;
+            break;
+        }
+        cnt = 3;
+    }
+
+    _080C1822:
```
Comment (jiangzhengwenjz) [inline src/worldmap_timemons.c:219]: unused label

File: src/worldmap_timemons.c [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+            *&y1 = proc->confs[i].node[gWMNodeData].y;
+
+            *&x2 = GM_SCREEN->x;
+            *&y2 = GM_SCREEN->y;
+        
+            x = x1 - x2;
+            y = y1 - y2 + 8;
+
+            if ((u16)y < 0xB0 && (u16)x < 0xF0)
+                proc->ap_procs[i] = APProc_Create(Sprite_08A97AEC, (s16)x, (s16)y, 0x3880, 0, 7);
```
Comment (jiangzhengwenjz) [inline src/worldmap_timemons.c:284]:
```suggestion
            if ((y >= 0 && y < 0xB0) && (x >= 0 && x < 0xF0))
                proc->ap_procs[i] = APProc_Create(Sprite_08A97AEC, x, y, 0x3880, 0, 7);
```

File: src/worldmap_timemons.c [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+     5,  5, 15, 20, 10, 20, 20,  5,  0,
+};
+
+const u8 WmMonsterGenerateRates_XmapEphraim[WM_MON_LOC_MAX] = {
+     5,  5, 15, 10, 15, 15, 15,  5, 15,
+};
+
+int GenerateRandomonsterMergeConf(int chapter, struct GmapTimeMonsConf * out)
+{
+    int r0, cnt, rn;
```
Comment (jiangzhengwenjz) [inline src/worldmap_timemons.c:153]:
```suggestion
    int cnt, rn;
```

**Review by jiangzhengwenjz — CHANGES_REQUESTED** [review]

File: src/worldmap_timemons.c (hunk 3 above)
Comment (MokhaLeee) [reply src/worldmap_timemons.c:153]: updated

**Pull request merged**